tests/jvm/src/test/scala/scala/meta/tests/trees/ReflectionSuite.scala

```diff
@@ -50,33 +51,36 @@ class ReflectionSuite extends FunSuite {
     // NOTE: we can't just do `duplicateRelevantFieldTpes.distinct`, because that doesn't account for `=:=`
     val distinctRelevantFieldTpes = ListBuffer[Type]()
     duplicateRelevantFieldTpes.foreach(tpe => if (!distinctRelevantFieldTpes.exists(_ =:= tpe)) distinctRelevantFieldTpes += tpe)
-    assert(distinctRelevantFieldTpes.sortBy(_.toString).mkString(EOL) === """
+    val obtained = distinctRelevantFieldTpes.sortBy(_.toString).mkString(EOL)
```

Comment:
This test now fails on 2.11, I propose we move this test to 2.12 only or we remove the test suite altoghther. wdyt @xeno-by ?

Reply:
Let's move to 2.12 only.

Reply:
Done.